Fixed to be able to set "personium.example.com" with Vagrantfile

Keita-Nakagama opened the pull request:
close #35

Keita-Nakagama committed:
Fixed to be able to set "personium.example.com" with Vagrantfile

KourokiTakuto commented on Vagrantfile:
```diff
@@ -66,11 +66,11 @@ Vagrant.configure(2) do |config|
+    config.proxy.https = "http://fj1738bt:0019970807@kmt.proxy.nic.fujitsu.com:8080"
+    config.proxy.no_proxy = "personium.example.com,127.0.0.1"
+  end
 
```
Please fix only "localhost".
Please check the difference before the pull request.

Keita-Nakagama replied:
Fixed it.

KourokiTakuto reviewed (commented):
Please correct personium-unit-config.properties.j2.

Keita-Nakagama committed:
Fixed only "localhost" of Vagrantfile.
Fixed "io.personium.core.engine.host=" in personium-unit-config.properties.j2 from "localhost" to "personium.example.com".

hiroaki-shibata reviewed (approved):
I confirmed and approve.
